static/src/javascripts/projects/commercial/modules/dfp/define-slot.js

```diff
@@ -40,7 +40,7 @@ const getSizeOpts = (sizesByBreakpoint: Object): Object => {
     const sizeMapping = buildSizeMapping(sizesByBreakpoint);
     // as we're using sizeMapping, pull out all the ad sizes, as an array of arrays
     const sizes = uniq(
-        flatten(sizeMapping, true, map => map[1]),
+        flatten(sizeMapping.map(size => size[1])),
```

Comment:
For sanity, does this look okay to everyone? Especially interested in the opinions of @guardian/commercial-dev 😄

Reply:
Good shout. Ads aren't appearing for me locally on this branch or on master. I'll test it on CODE next week. The input / output looks the same locally vs in production

Reply:
Looks good in CODE